x-pack/plugins/security_solution/public/security_integrations/cribl/components/lazy_custom_cribl_extension.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { PackagePolicyReplaceDefineStepExtensionComponent } from '@kbn/fleet-plugin/public/types';
+import { lazy } from 'react';
+import { CustomCriblForm } from './custom_cribl_form';
+
+export const LazyCustomCriblExtension = lazy<PackagePolicyReplaceDefineStepExtensionComponent>(
+  async () => {
+    return {
+      default: CustomCriblForm,
+    };
+  }
+);
```

Comment:
This is not lazy-loaded. Would it be possible to implement a real lazy import?
Something like this should work (untested):

```suggestion
import { lazy } from 'react';

export const LazyCustomCriblExtension = lazy<PackagePolicyReplaceDefineStepExtensionComponent>(
  async () => import('./custom_cribl_form').then(({ CustomCriblForm }) => ({ default: CustomCriblForm }))
);
```

Reply:
Updated!